### DISCUSSION TITLE
Option to disable labels in the taskbar

### DISCUSSION THREAD

**Discussion by CodeSpartan** [valinet/ExplorerPatcher#174] [Ideas]
I found myself enjoying the taskbar without any text in W11, it's much more efficient. Icons suffice, I just couldn't work with them grouped. Is it possible to use the W11 taskbar label-free icons using this patch?

**Comment by Gaurav-Original-ClassicShellTester** [comment]
You will have to use 7+ Taskbar Tweaker in combination with ExplorerPatcher to achieve this. See https://github.com/valinet/ExplorerPatcher/issues/9#issuecomment-939473986

**Reply by valinet** [reply]
I like how even people that want to see just the icons still hate the forced grouping 😂
